.changeset/mighty-beds-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Export `FavouriteEntity` component and icons
```

Comment:
A little 🇬🇧  sneaks in; up for renaming this one before exporting it? Should be `FavoriteEntity` to stick to `en-US` in the project. It should perhaps be moved to `catalog-react` too 🤔

Reply:
You mean catalogue-react right?

Reply:
![D0A3E7F8-5C71-465C-BCD8-23C6BD8306E7](https://user-images.githubusercontent.com/3097461/127456647-48f2b668-c4b2-46da-9da8-6144b4dc47ec.jpeg)